client/src/pages/App.js

```diff
@@ -119,36 +121,50 @@ const App = ({ pageDispatchers, pageProps }) => {
   const history = useHistory()
   const routerLocation = useLocation()
   const { loading, error, data, refetch } = API.useApiQuery(GQL_GET_APP_DATA)
+
   const { done, result } = useBoilerplate({
     loading,
     error,
     pageProps,
     pageDispatchers
   })
+  const skip = done || error || !data
+  const appState = skip ? null : processData(data)
+  const [
+    notifications,
+    loadingNotifications,
+    updateNotifications
+  ] = useNotifications(appState?.currentUser, skip)
```

Comment:
By retrieving the notifications here:
- first of all you retrieve the tasks the user is responsible for on when the App refreshes, the question is: do we want to do that here? What someone else is managing tasks and removes the user from a task and adds it to another task? That change would only be seen if the App refreshes, I don't know if that's ok.
- the same for the notifications, you determine them here, so changes to assessments would only be seen after an App refresh

Reply:
I wanted to retrieve responsible tasks in the GET_APP query, but couldn't without user uuid, so first need the user uuid to query. (for responsibleUserUuid). If that is something doable but that needs to be implemented at backend, i don't know.

Reply:
Added a new graphql query to fetch with APP_DATA